engine/frontend/src/component/ServiceInfo.js

```diff
@@ -30,32 +30,43 @@ const ServiceInfo = () => {
     const params = useParams()
     const {name: enclaveName, uuid: serviceUuid} = params;
 
-    useEffect(() => {
-        let stream;
-        const ctrl = new AbortController();
+    const startServiceLogCollector = (ctrl) => {
+        let stream
         const fetchLogs = async () => {
-            stream = await getServiceLogs(ctrl, enclaveName, serviceUuid, appData.apiHost);
+            stream = await getServiceLogs(ctrl, enclaveName, serviceUuid, appData.apiHost, true, 6000);
```

Comment:
maybe create constants for these? eg. `DEFAULT_SHOULD_FOLLOW_LOGS`, `DEFAULT_NUM_LOG_LINES`